### ISSUE TITLE
Rancher CLI - Incorrect help text seen when a user gives in an incorrect template version for a catalog app

### ISSUE THREAD

**Issue by sowmyav27** [rancher/rancher#23267]
**What kind of request is this (question/bug/enhancement/feature request):**

**Steps to reproduce (least amount of steps as possible):**
- Download Rancher CLI from rancher:v2.3.0-rc12
- Login to rancher using `./rancher login <rancher-server> --token <token>`
- Install an app using the below commands.

**$ ./rancher app install c-n76z2:test-2-fluentd-aggregator --version 0.2.1**
`FATA[0001] version 0.2.1 for template c-n76z2:test-2-fluentd-aggregator is invalid, run 'rancher show-template c-n76z2:test-2-fluentd-aggregator' for a list of versions` 

**$ ./rancher show-template c-n76z2:test-2-fluentd-aggregator**
`No help topic for 'show-template'`

**$ ./rancher app show-template c-n76z2:test-2-fluentd-aggregator**
```
0.1.0
0.1.2
0.1.6
0.3.0
```

**Expected Result:**
When user tries to install an app with incorrect version, user should be prompted with message:

FATA[0001] version 0.2.1 for template c-n76z2:test-2-fluentd-aggregator is invalid, run 'rancher **app** show-template c-n76z2:test-2-fluentd-aggregator' for a list of versions

**Other details that may be helpful:**

**Environment information**
- Rancher version (`rancher/rancher`/`rancher/server` image tag or shown bottom left in the UI): rancher:v2.3.0-rc12
- Installation option (single install/HA): HA

**Comment by deniseschannon** [comment]
Available in v2.4.0-rc3

**Comment by sowmyav27** [comment]
**Verified using rancher cli `v2.4.0-rc3`**

- Login to rancher using ./rancher login <rancher-server> --token <token>
- Install an app using the below commands. `./rancher app install fluentd-aggregator --version 0.2.1111` (give an invalid version of the app)
- The text in the error message is fixed. - `FATA[0000] version 0.2.1111 for template fluentd-aggregator is invalid, run 'rancher app show-template fluentd-aggregator' for a list of versions `

```
./rancher app install fluentd-aggregator --version 0.2.1111
FATA[0000] version 0.2.1111 for template fluentd-aggregator is invalid, run 'rancher app show-template fluentd-aggregator' for a list of versions 
$ ./rancher app show-template fluentd-aggregator
0.1.0
0.2.1
0.3.0
0.3.1
```
